client/app/routes/application.js

```diff
@@ -25,5 +26,10 @@ export default Ember.Route.extend(ApplicationRouteMixin, {
     } else if (this.get('currentDistrict.isInvalid')) {
       this.replaceWith('index');
     }
+  },
+  actions: {
+    trackEvent(eventName) {
+      this.get('intercom').trackEvent(eventName);
+    }
```

Comment:
I think the difference in how we handled `boot` and `shutdown` is that the service injected in the action hash in the application route exposes it to the entire app so we can access it many times in any template file. I think that is something that may make trackEvent different than the other two actions.